libnavigation-base/src/main/java/com/mapbox/navigation/base/route/RouteAlternativesOptions.kt

```diff
@@ -62,30 +70,43 @@ class RouteAlternativesOptions private constructor(
      */
     class Builder {
         private var intervalMillis: Long = TimeUnit.MINUTES.toMillis(5)
+        private var avoidManeuverSeconds = 1
```

Comment:
let's set to `8`, it's recomended value 
```suggestion
        private var avoidManeuverSeconds = 8
```

Reply:
where is it recommended? hardcoded value was `1`, NN has `0` as  default

Reply:
it's the recommendation of automotive [doc](https://docs.google.com/document/d/1y3Aqj3I5rfLBHyyfusO9RwV8lH0G4ckMI_xJKBLfFts/edit#)
```
How should it be implemented?
We will use the speed of the edge to determine the vehicle speed. The next maneuver must not be earlier than 8s ahead (best case: configurable) based on the current speed.
```